index.js

```diff
@@ -6,10 +6,12 @@ function createFunction(parameters, defaultType) {
     if (!isFunctionDefinition(parameters)) {
         fun = function() { return parameters; };
         fun.isFeatureConstant = true;
-        fun.isGlobalConstant = true;
+        fun.isZoomConstant = true;
 
     } else {
-        var property = parameters.property === undefined ? '$zoom' : parameters.property;
+        var zoomAndDataDependent = typeof parameters.stops[0][0] === 'object';
+        var dataDependent = zoomAndDataDependent || parameters.property !== undefined;
```

Comment:
Sounds good. Will this also apply to how we talk about these things externally? Which do you prefer?